### PR TITLE
Update Rabbit config for local set-up

### PR DESCRIPTION
**Pull request by aef-**
I ran into some insidious issues with a new set-up. RabbitMQ would timeout/crash when a celery, or any process, attempted to connect. That lead me to: https://stackoverflow.com/questions/38343656/epmd-error-for-host-myhost-address-cannot-connect-to-host-port-on-windows-10

The errors:
`ERROR: epmd error for host <MY HOSTNAME>: timeout (timed out)`

```
> CRASH REPORT Process <0.272.0> with 0 neighbours exited with reason: no match of right hand value noport in rabbit_networking:record_distribution_listener/0 line 305 in application_master:init/4 line 138
2020-05-12 14:26:48.124 [info] <0.43.0> Application rabbit exited with reason: no match of right hand value noport in rabbit_networking:record_distribution_listener/0 line 305
```

### CODE REVIEW

**Comment by aef-** [comment]
Will this impact Staging/Prod?

**Comment by allanbolipata** [comment]
Since this is self-contained in the Makefile, it doesn't look like this will affect staging/production

**Comment by allanbolipata** [comment]
@aef- Since #208 hasn't been fixed, will move to `1.7.0` milestone

File: Makefile [hunk 1]
```diff
@@ -190,9 +190,9 @@ export BEAGLE_JOB_SCHEDULER_QUEUE:=beagle_job_scheduler_queue
 # corresponds to ./conf/rabbitmq.conf ;
 export RABBITMQ_CONFIG_FILE:=$(CURDIR)/conf/rabbitmq
 # give the RabbitMQ node cluster a name based on current dir; hopefully different from other instances on same server
-export RABBITMQ_NODENAME:=rabbit_$(CURDIR_BASE)
-export RABBITMQ_NODE_IP_ADDRESS:=127.0.0.1
```
[the remark this answers is not in the document]
Comment (aef-) [reply Makefile:194]: updated, let me know if that works for you!

File: Makefile [hunk 2]
```diff
@@ -320,6 +320,7 @@ django-init:
 	python manage.py migrate
 	python manage.py createsuperuser
 	python manage.py loaddata \
+	beagle_etl.operator.json \
```
[the remark this answers is not in the document]
Comment (aef-) [reply Makefile:323]: I believe so!

**Comment by aef-** [comment]
Hey anything I can do to get this merged?

**Comment by stevekm** [comment]
@allanbolipata we should look at this again and see if it can be merged

**Review by sivkovic — APPROVED** [review]

**Pull request merged**